config/split_mlog_adapter.go

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package config
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-server/v5/mlog"
+)
+
+type splitMlogAdapter struct {
+	wrappedLog *mlog.Logger
+}
+
+func (s *splitMlogAdapter) Error(msg ...interface{}) {
+	s.wrappedLog.Error(fmt.Sprint(msg...))
+}
+
+func (s *splitMlogAdapter) Warning(msg ...interface{}) {
+	s.wrappedLog.Warn(fmt.Sprint(msg...))
+}
+
+// Ignoring more verbose messages from split
+func (s *splitMlogAdapter) Info(msg ...interface{}) {
+	//s.wrappedLog.Info(fmt.Sprint(msg...))
+}
+
+func (s *splitMlogAdapter) Debug(msg ...interface{}) {
+	//s.wrappedLog.Debug(fmt.Sprint(msg...))
+}
+
+func (s *splitMlogAdapter) Verbose(msg ...interface{}) {
+	//s.wrappedLog.Info(fmt.Sprint(msg...))
+}
```

Comment:
Do we maybe want to dump full split log messages to a file or something in the future? I can see how they would get messy, but may be nice to have them out of band somehow if something funky happens. Files aren't great for a cloud/container environment, but just wanted to note another option.

Just a thought.

Reply:
I was originally going to just tag the messages and set the level, but the messages that split logs are super verbose and unhelpful. Hence the config setting and the commented out levels.